### PR TITLE
+libres3 1.1

### PR DESCRIPTION
**Pull request by edwintorok**
not available on OCaml 4.02.2 due to http://caml.inria.fr/mantis/view.php?id=6919

### CODE REVIEW

**Comment by edwintorok** [comment]
two of the ubuntu builders failed to trouble with mirrors, how do I tell them to rebuild?

File: packages/libres3/libres3.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
+opam-version: "1.2"
+maintainer: "dev-team@skylable.com"
+authors: [ "edwin@skylable.com" ]
+license: "GPL-2.0 with OpenSSL exception"
+homepage: "http://www.skylable.com/products/libres3"
+bug-reports: "https://bugzilla.skylable.com/"
+dev-repo: "http://git.skylable.com/libres3"
```
Comment (avsm) [inline packages/libres3/libres3.1.1/opam]: i think this needs a `.git` extension in the URL for dev-repo

Comment (edwintorok) [inline packages/libres3/libres3.1.1/opam]: actually both work, but I'll add the `.git`

**Comment by dsheets** [comment]
Would be nice to make `dev-repo` https in future.

**Comment by edwintorok** [comment]
Thanks. I've [opened a bug](https://bugzilla.skylable.com/show_bug.cgi?id=1416), although we do have gpg signature on tags.

**Pull request merged**